openshift-ci-debugging/login_to_ci_cluster

```diff
@@ -5,7 +5,7 @@ E2E_REGEX="e2e-\S*e2e"
 IPI_REGEX="install-\S*install"
 
 # Defined Commands
-REGEX_CMD="grep -P"  # TODO: handle Mac shells, where is "grep -p"
+REGEX_CMD="grep -P"
```

Comment:
in this case, all of the regex used here is ERE-compatible/doesn't need PCRE.  Given that, should we just change this to `grep -E` since that works on all platforms, and remove the os typecatch below?

Reply:
Done.